modules/tile-converter/src/i3s-converter/helpers/node-index-document.ts

```diff
@@ -90,6 +102,9 @@ export class NodeIndexDocument {
    * Add neighbors to child nodes of this node
    */
   public async addNeighbors(): Promise<void> {
+    if (this.finalized) {
```

Comment:
Is this supposed to be called many times, or is this an error check? If the latter, throw an error instead?